app/lib/actions/middleware/watch_delegated_proxy_sub_tasks.rb

```diff
@@ -2,13 +2,13 @@ module Actions
   module Middleware
     class WatchDelegatedProxySubTasks < ::Dynflow::Middleware
       class CheckOnProxyActions; end
-      POLL_INTERVAL = 30
+      POLL_INTERVAL = 10
```

Comment:
Was it intentional to shorten the interval? I would actually expect to prolong this a bit, perhaps even something like once per 5 minutes could be fine.